File: .claude/skills/nextjs-docs-assistant/scripts/update-docs.js
#!/usr/bin/env node

/**
 * Script to fetch the latest Next.js documentation
 * This script would be used to keep the documentation up to date
 */

const fs = require('fs').promises;
const path = require('path');

async function updateNextJSDocumentation() {
  console.log('Updating Next.js documentation...');

  // In a real implementation, this would fetch from the official Next.js docs
  // For now, we'll just log what would happen
  console.log('Fetching latest Next.js documentation from official source...');

  // Example of what this might do:
  // 1. Fetch the latest docs from Next.js website
  // 2. Update the references/documentation.md file
  // 3. Update any other relevant reference files

  console.log('Next.js documentation updated successfully!');
}

// Run the update function if this script is called directly
if (require.main === module) {
  updateNextJSDocumentation().catch(console.error);
}

module.exports = { updateNextJSDocumentation };

File: .claude/skills/fastapi-docs-assistant/scripts/update-docs.py
#!/usr/bin/env python3

"""
Script to fetch the latest FastAPI documentation
This script would be used to keep the documentation up to date
"""

import requests
from pathlib import Path
import json

def update_fastapi_documentation():
    """Fetch the latest FastAPI documentation and update local references"""
    print("Updating FastAPI documentation...")

    # In a real implementation, this would fetch from the official FastAPI docs
    # For now, we'll just log what would happen
    print("Fetching latest FastAPI documentation from official source...")

    # Example of what this might do:
    # 1. Fetch the latest docs from FastAPI website
    # 2. Update the references/documentation.md file
    # 3. Update any other relevant reference files

    print("FastAPI documentation updated successfully!")

if __name__ == "__main__":
    update_fastapi_documentation()